docs/src/main/markdown/json.md

```diff
@@ -15,8 +15,17 @@ Make sure that you are using ScalaPB 0.5.x or later.
 In `build.sbt` add a dependency on `scalapb-json4s`:
 
 ```scala
+// For ScalaPB 0.10.x:
+libraryDependencies += "com.thesamet.scalapb" %% "scalapb-json4s" % "0.10.0-M1"
```

Comment:
I am guessing we want to want for a `0.10.1` to merge to avoid the confusion that `0.10.0-M1` follows `0.10.0` ?

Reply:
Agreed. I just released 0.10.1-M1 which is otherwise identical to 10.0.0-M1, to avoid the confusion. Can you update?

Reply:
force-pushed